deploy/helm/proto/collector/logs/v1/logs_service.proto

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020, OpenTelemetry Authors
```

Comment:
instead of adding those files individually, can you set whole repo https://github.com/open-telemetry/opentelemetry-proto as submodule (it could be in the same folder `deploy/helm/proto`)

Reply:
Changed as suggested.